proposals/0000-style-script-defaults.md

```diff
@@ -0,0 +1,88 @@
+- Start Date: 2022-03-15
+- Reference Issues: [RFC0008](https://github.com/withastro/rfcs/tree/main/proposals/0008-style-script-behavior.md), [#65](https://github.com/withastro/rfcs/discussions/65)
+- Implementation PR: <!-- leave empty -->
+
+# Summary
+
+Astro is inconsist between `<style>` and `<script>` default behavior. Currently, `<style>` has opt-out processing, but `<script>` does not. This RFC aims to settle on a good, consistent default for both `<style>` and `<script>`.
+
+- New `is:inline` directive to avoid bundling `style` or `script`
+- `<style>` => remains scoped, bundled by default. `is:scoped` directive supported for consistency
```

Comment:
Are we preserving the new behavior added by @FredKSchott that imports inside of style are not scoped? What happens if you add `is:scoped` explicitly, those are still not scoped?

Reply:
No change here, so yes that behavior is preserved.